rosdep/base.yaml

```diff
@@ -4090,6 +4090,7 @@ libtiff-dev:
     artful: [libtiff5-dev]
     bionic: [libtiff5-dev]
     lucid: [libtiff4-dev]
+    focal: [libtiff-dev]
```

Comment:
libtiff5-dev?

That was the one using in bionic, so I don't think we should pick an older version in focal.

Reply:
I think libtiff5-dev is a transitional package; i.e. it's a dummy package that points to libtiff-dev: https://packages.ubuntu.com/focal/libtiff5-dev

Reply:
good point